src/common/interfaces.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <interfaces/echo.h>
+#include <interfaces/handler.h>
+
+#include <boost/signals2/connection.hpp>
+#include <utility>
+#include <memory>
```

Comment:
pico-nit: `clang-format-diff.py` complains about header order:
```suggestion
#include <memory>
#include <utility>
```

Reply:
> pico-nit: `clang-format-diff.py` complains about header order:

Thanks, fixed